### ISSUE TITLE
Create YarpOpenraveAnalogSensors

### ISSUE THREAD

**Issue by jgvictores** [roboticslab-uc3m/openrave-yarp-plugins#49]
Follow-up from #33, on current `develop`: create `YarpOpenraveAnalogSensors` similar to `YarpOpenrraveGrabber`, also based on contents that can be sliced from `YarpOpenraveCoupled`. Can then erase `OpenraveYarpCoupled` as all functionalities implemented in a modular fashion.

**Comment by jgvictores** [comment]
Blocked by #50.

**Comment by jgvictores** [comment]
#50 is no longer blocking, as there is already `YarpOpenraveBase` for code reuse. New code reuse can be documented/consulted there if required.

**Comment by jgvictores** [comment]
https://github.com/robEllenberg/forceSensor/issues/1 -> https://github.com/roboticslab-uc3m/forceSensor

**Comment by jgvictores** [comment]
http://www.yarp.it/group__dev__iface__multiple__analog.html

**Comment by jgvictores** [comment]
http://www.yarp.it/classyarp_1_1dev_1_1ISixAxisForceTorqueSensors.html

**Comment by jgvictores** [comment]
https://github.com/robotology/yarp/issues/1526

**Comment by jgvictores** [comment]
https://github.com/robotology/icub-main/tree/master/src/libraries/icubmod/embObjFTsensor

**Comment by jgvictores** [comment]
Note: `sensorIndex` -> `sensorIndices`

**Comment by jgvictores** [comment]
Renamed issue again: an IMU sensor would go here too

**Comment by jgvictores** [comment]
No longer `upstream`: https://github.com/robotology/yarp/issues/2154 is on the way via https://github.com/robotology/yarp/pull/2158

**Comment by jgvictores** [comment]
ToDo (as of https://github.com/roboticslab-uc3m/openrave-yarp-plugins/compare/c26c9ce4cd4bf89bdd5296e731062ded276ba1ec...ad85f4aa3a514427bb342f8d460eaa5cb080875d):
- [x] Extend to multiple sensors
- [x] Refs to https://github.com/roboticslab-uc3m/forceSensor (sensor plugin + example)
- [x] Spawn new issue on IMU

**Comment by jgvictores** [comment]
> * [x]  Extend to multiple sensors

Done at https://github.com/roboticslab-uc3m/openrave-yarp-plugins/commit/a075af5e2691da0d2c59737159b2a5e412625447

**Comment by jgvictores** [comment]
> * [x]  Refs to https://github.com/roboticslab-uc3m/forceSensor (sensor plugin + example)

Done at https://github.com/roboticslab-uc3m/openrave-yarp-plugins/commit/889059cc59d4e3776091b03357e0af78c6d84507

**Comment by jgvictores** [comment]
> * [x]  Spawn new issue on IMU

Created https://github.com/roboticslab-uc3m/openrave-yarp-plugins/issues/110

**Comment by jgvictores** [comment]
Next up: nuke [OpenraveYarpCoupled](https://github.com/roboticslab-uc3m/openrave-yarp-plugins/tree/d666b827a8ebb6e119219769712a42d5429f8fe0/libraries/OpenravePlugins/OpenraveYarpCoupled), which is related to "Slice non-force from OpenraveYarpForce and rename openrave-YarpForce.py" (https://github.com/roboticslab-uc3m/openrave-yarp-plugins/issues/33) from where this issue spawned.

Note that it was used in textiles:
- https://github.com/roboticslab-uc3m/teo-configuration-files/issues/26 points to https://github.com/roboticslab-uc3m/textiles/tree/b2862bb5ea61d07ea6f3c593fb984a55cb30e3a7/textiles/ironing/manipulation
- Moreover, see https://github.com/roboticslab-uc3m/openrave-yarp-plugins/blob/d666b827a8ebb6e119219769712a42d5429f8fe0/examples/python/openraveYarpCoupled.py#L14

**Comment by jgvictores** [comment]
> Next up: nuke [OpenraveYarpCoupled](https://github.com/roboticslab-uc3m/openrave-yarp-plugins/tree/d666b827a8ebb6e119219769712a42d5429f8fe0/libraries/OpenravePlugins/OpenraveYarpCoupled), which is related to "Slice non-force from OpenraveYarpForce and rename openrave-YarpForce.py" (#33) from where this issue spawned.

Done at https://github.com/roboticslab-uc3m/openrave-yarp-plugins/commit/6da904562d9fe84787f7213d1dc5856575cf5112

Next up: add mention to MAS and YARP version.

**Comment by jgvictores** [comment]
> Next up: add mention to MAS and YARP version.

Done at https://github.com/roboticslab-uc3m/openrave-yarp-plugins/commit/9bd4adb2e9a45fa2a64343c46e4302df1cc29fbb

**Comment by jgvictores** [comment]
Merged at https://github.com/roboticslab-uc3m/openrave-yarp-plugins/commit/1b23ca0969ac1160040d8e557c9b169ddb62b699
